subprojects/signing/src/integTest/groovy/org/gradle/plugins/signing/SigningPublicationsIntegrationSpec.groovy

```diff
@@ -541,10 +576,6 @@ class SigningPublicationsIntegrationSpec extends SigningIntegrationSpec {
                     }
                     ivy {
                         url "file://\$buildDir/ivyRepo/"
-                        patternLayout {
```

Comment:
❌ It seems the only reason this had to change was the change to `ivyRepoFile`. I wouldn't change it.